### ISSUE TITLE
Import path resolution broken on Windows

### ISSUE THREAD

**Issue by RIAstar** [brillout/gulp-jspm#34]
I have a build that fails on Windows but succeeds on Linux.
The main file is loaded correctly, but after that all the paths of the imports are relative to the `baseURL` while they should be relative to the main file.

Example error:
```
gulp-jspm: Building the single-file sfx bundle for `build\app\App.js`...
gulp-jspm: Error: Error on fetch for _module.js at file:///c:/path/to/my-app/_module.js
        Loading build\app\App.js                                                                                                     
        ENOENT: no such file or directory, open 'c:\path\to\my-app\_module.js'         
```
Whereas it should be looking for `c:\path\to\my-app\build\app\_module.js`

I checked whether this was an issue in JSPM itself:
Running `jspm bundle-sfx build\\app\\App.js` on the command line fails as well, whereas `jspm bundle-sfx build/app/App.js` succeeds.
After digging through the JSPM issues, I found that [bundling works in URL space](https://github.com/jspm/jspm-cli/issues/1919#issuecomment-227005298). 

In other words: the path to the input file should always use forward slashes, no matter the OS.
Easily fixed by tacking `replace('\\', '/')` onto https://github.com/brillout/gulp-jspm/blob/master/index.js#L119

**Comment by brillout** [comment]
thanks! See https://github.com/brillout/gulp-jspm/commit/026aa95b611d9d205dce18114491d140303f598c

**Comment by RIAstar** [comment]
That's fast! And published already.

Just verified: works like a charm.
Thanks.

**Comment by brillout** [comment]
thanks for the research!
